### DISCUSSION TITLE
How can i consume the Git Hub API

### DISCUSSION THREAD

**Discussion by InacioCarvalhoOliveira** [community/community#34518] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello @Everyone !!

I want to filter User names, most used  programing languages and projects names on my Web Site using the GitHub API, is this possible ?

**Comment by jfullstackdev** [comment, marked as answer]
not exactly GitHub API but maybe this

https://github.com/madnight/githut

and since that's open source, you don't need to start from scratch, you just need to collaborate or even keep a version of yours if the license will allow it
